docs/trace/getting-started-jaeger/getting-started-jaeger.csproj

```diff
@@ -1,7 +1,14 @@
 <Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <Frameworks>|net462|net472|net48|</Frameworks>
```

Comment:
Might be an overkill?
There seems to be some prior arts https://github.com/open-telemetry/opentelemetry-dotnet/blob/6c73e368085f11b8b3478e1ff9ebdaaa5f2b1c36/test/OpenTelemetry.Instrumentation.Grpc.Tests/OpenTelemetry.Instrumentation.Grpc.Tests.csproj.

Reply:
Removed the `<PropertyGroup>` and added a `PackageReference` with condition based on the `Grpc.Tests.csproj`.